src/pages/settings/Subscription/SubscriptionPlan.tsx

```diff
@@ -20,9 +23,12 @@ function SubscriptionPlan() {
     const styles = useThemeStyles();
     const theme = useTheme();
 
-    const subscriptionPlan = useSubscriptionPlan();
     const [privateSubscription] = useOnyx(ONYXKEYS.NVP_PRIVATE_SUBSCRIPTION);
 
+    const subscriptionPlan = useSubscriptionPlan();
+    const subscriptionPrice = useSubscriptionPrice();
```

Comment:
Just a thought, but since `useSubscriptionPrice` also calls `useSubscriptionPlan` and `usePreferredCurrency` inside it, we end up having the double of Onyx connections. It could or could not be an issue, but let's keep this in mind.

Reply:
Do you have any suggestions on how to improve it?